sockeye/inference.py

```diff
@@ -452,31 +460,40 @@ class ModelState:
     A ModelState encapsulates information about the decoder states of an InferenceModel.
     """
 
-    def __init__(self, states: List[mx.nd.NDArray]) -> None:
+    def __init__(self, states: List[mx.nd.NDArray]=None) -> None:
         self.states = states
 
-    def sort_state(self, best_hyp_indices: mx.nd.NDArray):
+    def sort_state(self, best_hyp_indices: mx.nd.NDArray) -> None:
```

Comment:
This function does not return None

Reply:
Yeah, was copied from __init__() which had -> None for some reason

Reply:
the reason we annotate the return type of constructors as None is to make mypy happy :)
For other functions its not needed